### PR TITLE
`show-names` - Suppot new commit view

### PR DESCRIPTION
**Pull request by kovsu**
- Part of #7808 

## Test URLs

https://github.com/refined-github/sandbox/commit/54c10fe670779eab43b6a18b7fb06e51dd7050af#r148774860

## Screenshot

image

### CODE REVIEW

**Comment by kovsu** [comment]
Actually it doesn't show current username here, but for testing I let it shows here.

File: source/github-helpers/selectors.ts [hunk 1]
```diff
@@ -196,6 +196,7 @@ const authorLinks = [
 	'.inline-comments a.author',
 	'h3 a[data-testid="issue-body-header-author"]', // The first issue comment
 	'.react-issue-comment a[data-testid="avatar-link"]',
+	'[data-testid="comment-header"] a[data-testid="avatar-link"]',
```
Comment (fregante) [inline source/github-helpers/selectors.ts:199]: // where?

**Comment by fregante** [comment]
You can add labels to PRs now 🙌

File: source/github-helpers/selectors.ts (hunk 1 above)
Comment (kovsu) [reply source/github-helpers/selectors.ts:199]: Oh, I remember I added it.

**Review by fregante — APPROVED** [review]

**Review by fregante — APPROVED** [review]

**Pull request merged**